### PR TITLE
Pre-release editing

### PR DESCRIPTION
**Pull request by JayH5**

### CODE REVIEW

File: docs/getting-started.rst [hunk 1]
```diff
@@ -62,9 +62,11 @@ A few things to note here:
   the test begins and that the container is stopped and removed after the test
   ends.
 - The scope of the fixture is important. By default, pytest fixtures have
-  function scope, which means for each test function the fixture is completely
-  reinitialized. Creating and starting up a container can be a little slow, so
-  you need to think carefully about what scope to use for your fixtures.
+  function scope, which means that for each test function the fixture is
+  completely reinitialized. Creating and starting up a container can be a
+  little slow, so you need to think carefully about what scope to use for your
+  fixtures. See :meth:`~seaworthy.definitions.ContainerDefinition.clean` for a
```
Comment (jerith) [inline docs/getting-started.rst]: Does this render as `ContainerDefinition.clean` or just `clean`? If the latter, we may need to do something fancier with the reference text.

File: seaworthy/pytest/fixtures.py [hunk 2]
```diff
@@ -14,7 +14,7 @@ def docker_helper_fixture(name='docker_helper', scope='module', **kwargs):
     Create a fixture for :class:`~seaworthy.DockerHelper`.
 
     This can be used to create a fixture with a different name to the default.
-    It can also be used to override the scope of the default fixture:::
+    It can also be used to override the scope of the default fixture::
```
Comment (jerith) [inline seaworthy/pytest/fixtures.py]: `T_PAAMAYIM_NEKUDOTAYIM`! :trollface:

File: seaworthy/pytest/fixtures.py [hunk 3]
```diff
@@ -54,8 +54,9 @@ def resource_fixture(definition, name, scope='function'):
     """
     Create a fixture for a resource.
 
-    Note that it is important to keep a reference to the fixture function
-    returned by this function::
+    Note that this function returns a fixture function. It is important to keep
+    a reference to the returned function within the scope of the tests that use
+    the fixture::
```
Comment (jerith) [inline seaworthy/pytest/fixtures.py]: Should this (and others like it) be a `.. note::` block?

Comment (JayH5) [reply seaworthy/pytest/fixtures.py]: I did not know about this. Giving it a go.

File: docs/getting-started.rst (hunk 1 above)
Comment (JayH5) [reply docs/getting-started.rst]: The latter :(

File: seaworthy/pytest/fixtures.py (hunk 2 above)
Comment (JayH5) [reply seaworthy/pytest/fixtures.py]: 😱 wow

File: seaworthy/pytest/fixtures.py [hunk 4]
```diff
@@ -97,25 +98,27 @@ def clean_container_fixtures(container, name, scope='class'):
...
-                Test something about the container that doesn't require it to
-                have a clean state (e.g. testing something about a dependent
-                container).
-                \"""
+    .. note:: This function returns two fixture functions. It is important to
+        keep references to the returned functions within the scope of the tests
+        that use the fixtures::
+
+            f1, f2 = clean_container_fixtures(
+                PostgreSQLContainer(), 'postgresql')
```
Comment (JayH5) [inline seaworthy/pytest/fixtures.py]: Having to split this over two lines is a pain and makes the generated docs look not-so-good...but I'm not sure how else to appease flake8's line-length check.

File: seaworthy/pytest/fixtures.py (hunk 2 above)
Comment (jerith) [reply seaworthy/pytest/fixtures.py]: I tried to find an appropriate translation of "thrice colon" ("pa'amayim nekudotayim" is the transliteration of "פַּעֲמַיִם נְקוּדָתַיִם" which translates to "twice colon") to put in there, but my Hebrew isn't good enough to keep the same form without turning it into "three colon" or "colon three times" and neither of those are sufficiently recognisable for the troll to work.

File: seaworthy/pytest/fixtures.py (hunk 4 above)
Comment (jerith) [reply seaworthy/pytest/fixtures.py]: Maybe keep the example outside the `note` block?

**Pull request merged**